pam/pam_adsys.c

```diff
@@ -215,6 +216,44 @@ static int update_machine_policy(pam_handle_t * pamh, int debug)
 	return PAM_SYSTEM_ERR;	/* will never be reached. */
 }
 
+/*
+ * Get default domain suffix from SSSD_CONF_PATH
+ */
+static char* get_default_domain(pam_handle_t* pamh) {
+	FILE* f = fopen(SSSD_CONF_PATH, "r");
+	if (f == NULL) {
+		pam_syslog(pamh, LOG_DEBUG, "Failed to open sssd.conf");
+		return "\0";
+	}
+
+	size_t buffsize = 256;
+	char* line = malloc(sizeof(char) * buffsize);
+	char* domain = "\0";
+	while (getline(&line, &buffsize, f) != -1) {
+		if (strncmp(line, "default_domain_suffix", 21) == 0) {
```

Comment:
Are you sure we can’t have a valid:

`default_domain_suffix = `

or even:

` default_domain_suffix=`

?

Reply:
I’m unsure the first one will work, as the character index will be 22 and not 21.

Do you mind trying both in your setup?

Reply:
I was always using it as `default_domain_suffix = domain`